_sass/minima/custom-styles.scss

```
@@ -5,6 +5,7 @@ body {
   font-weight: lighter;
   color: $color--white;
   line-height: 1.8;
+  font-family: Verdana;
```

It is safer to add font fallback stack :

```suggestion
  font-family: "Verdana", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
```